cou/apps/base.py

```diff
@@ -697,5 +697,5 @@ def _get_wait_step(self) -> PostUpgradeStep:
         return PostUpgradeStep(
             description=description,
             parallel=False,
-            coro=self.model.wait_for_idle(self.wait_timeout, apps=apps),
+            coro=self.model.wait_for_active_and_idle(self.wait_timeout, apps=apps),
```

Comment:
IMO shorter name is better and `wait_for_active` should be enough, since `active` means that it's idle. Or we can do at least `wait_for_active_idle`. What do you say?

Reply:
I always prefer been explicit so, `wait_for_active_idle` sounds better to me. I'm going to change it